<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewAceDialog</class>
 <widget class="QDialog" name="NewAceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>309</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>787</width>
    <height>309</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>787</width>
    <height>309</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>763</width>
     <height>277</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="ownerNameText">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Owner:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ownerNameLine">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>21</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="aceTypeText">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>23</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>ACE Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="aceTypeBox">
        <property name="minimumSize">
         <size>
          <width>191</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>191</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>761</width>
        <height>181</height>
       </size>
      </property>
      <property name="title">
       <string>Access Rights</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>748</width>
         <height>156</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="GENERIC_READ_">
              <property name="text">
               <string>GENERIC_READ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="GENERIC_WRITE_">
              <property name="text">
               <string>GENERIC_WRITE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="GENERIC_EXECUTE_">
              <property name="text">
               <string>GENERIC_EXECUTE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="GENERIC_ALL_">
              <property name="text">
               <string>GENERIC_ALL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="DELETE_">
              <property name="text">
               <string>DELETE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="READ_CONTROL_">
              <property name="text">
               <string>READ_CONTROL</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="WRITE_DAC_">
              <property name="text">
               <string>WRITE_DAC</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="WRITE_OWNER_">
              <property name="text">
               <string>WRITE_OWNER</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="SYNCHRONIZE_">
              <property name="text">
               <string>SYNCHRONIZE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="STANDARD_RIGHTS_REQUIRED_">
              <property name="text">
               <string>STANDARD_RIGHTS_REQUIRED</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="STANDARD_RIGHTS_ALL_">
              <property name="text">
               <string>STANDARD_RIGHTS_ALL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_OPEN_">
              <property name="text">
               <string>ACTRL_DS_OPEN</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_CREATE_CHILD_">
              <property name="text">
               <string>ACTRL_DS_CREATE_CHILD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_DELETE_CHILD_">
              <property name="text">
               <string>ACTRL_DS_DELETE_CHILD</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_LIST_">
              <property name="text">
               <string>ACTRL_DS_LIST</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_READ_PROP_">
              <property name="text">
               <string>ACTRL_DS_READ_PROP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_WRITE_PROP_">
              <property name="text">
               <string>ACTRL_DS_WRITE_PROP</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_SELF_">
              <property name="text">
               <string>ACTRL_DS_SELF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_DELETE_TREE_">
              <property name="text">
               <string>ACTRL_DS_DELETE_TREE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_LIST_OBJECT_">
              <property name="text">
               <string>ACTRL_DS_LIST_OBJECT</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="ACTRL_DS_CONTROL_ACCESS_">
              <property name="text">
               <string>ACTRL_DS_CONTROL_ACCESS</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
